gremlin-driver/pom.xml

```
@@ -253,6 +253,18 @@ limitations under the License.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
```

```suggestion
                <configuration>
```
Try removing these extra configs and see if this still works. The plugin version and source/target config should be inherited from the root pom.xml (https://github.com/apache/tinkerpop/blob/097b32e9171164048e0e5334d83afe329eed1775/pom.xml#L556C13-L556C13, https://github.com/apache/tinkerpop/blob/097b32e9171164048e0e5334d83afe329eed1775/pom.xml#L939C28-L939C28).

In general it's best not to override these settings unless strictly necessary.